libcxx/test/std/input.output/iostreams.base/ios.base/ios.types/ios_Init/ios_Init.global.pass.cpp

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <iostream>
+
+extern "C" const char* __asan_default_options() { return "check_initialization_order=true:strict_init_order=true"; }
```

Comment:
Can we just change the default options in the CI?

Reply:
Will not be enough. Tried on local build. Somewhere in the middle libc++ lit setup filters out "env".
Other components allow ASAN stuff pass through, but, last time I checked, not libc++.

I don't think we can do that in this patch without additional testing. It may require fixing some tests, as out bots run with env with stricter set of options.

Maybe we can file a new issue, and look into that later? I would not be able to do so in the next couple of weeks.

Would you prefer land this as is, and remove __asan_default_options later, or remove now and expect proper env  in future?

Reply:
I think I'd like an issue that the libc++ ASan CI should enable more (or all?) optional checks and adding a link to it in the test.